### PR TITLE
Presentation based on P3042 with extras based on proposal progress

### PR DESCRIPTION
**Pull request by Twon**

### CODE REVIEW

**Comment by Twon** [comment]
@jbcoe this is just in a separate document for now. Let me know if you think this should be merged into a wider "technical discussion" document or something else.

File: presentations/2023-11-27-bsi-value-types.md [hunk 1]
```diff
@@ -256,6 +256,60 @@ Both of the proposed types, `indirect` and `polymorphic`, need:
 
 ---
 
+# Semantially composition with `std::unique_ptr`
```
Comment (jbcoe) [inline presentations/2023-11-27-bsi-value-types.md:259]: Typo here I think. Not sure what Semantially should be

File: presentations/2023-11-27-bsi-value-types.md [hunk 2]
```diff
@@ -330,14 +384,20 @@ int main ()
 
 ---
 
-# Further details
+ # Further details
```
Comment (jbcoe) [inline presentations/2023-11-27-bsi-value-types.md:387]: Remove extra space

**Review by jbcoe — DISMISSED** [review]

File: presentations/2023-11-27-bsi-value-types.md (hunk 1 above)
Comment (Twon) [reply presentations/2023-11-27-bsi-value-types.md]: Updates to `Semantically`.  Let me know if you see a better wording here.

**Review by jbcoe — APPROVED** [review]

**Pull request merged**